scripts/new_project.kts

```diff
@@ -46,6 +48,22 @@ object NewProject {
             return@copyRecursively OnErrorAction.TERMINATE
         }
     }
+
+    private fun cleanNewProjectFolder() {
+        executeCommand("./../$appNameWithoutSpace/gradlew -p ../$appNameWithoutSpace clean")
+        executeCommand("./../$appNameWithoutSpace/gradlew -p ../$appNameWithoutSpace/buildSrc clean")
+        File("../$appNameWithoutSpace/.idea")?.deleteRecursively()
+        File("../$appNameWithoutSpace/.gradle")?.deleteRecursively()
+        File("../$appNameWithoutSpace/buildSrc/.gradle")?.deleteRecursively()
+        File("../$appNameWithoutSpace/.git")?.deleteRecursively()
+    }
+
+    private fun executeCommand(command: String) {
+        val process = Runtime.getRuntime().exec(command)
+        InputStreamReader(process.inputStream).forEachLine {
+            println(it)
+        }
```

Comment:
small improvement:
```suggestion
        val process = Runtime.getRuntime().exec(command)
        process.inputStream.reader().forEachLine {
            println(it)
        }
```

Reply:
Nice, maybe we don't need the import `import java.io.InputStreamReader` then